### ISSUE TITLE
Some member of cython cannot be known by pyright

### ISSUE THREAD

**Issue by Freed-Wu** [microsoft/pyright#9411]
```python
import cython
```

press `cython.`, it will complete cythons' member. However, many are missing. such as `cython.cfunc`.

![Image](https://github.com/user-attachments/assets/316f9cf8-4c7b-42b3-86cd-fefb1e5f0a53)

`Cython/Shadow.py`:

```python
cclass = ccall = cfunc = _EmptyDecoratorAndManager()
```

`cython.py`:

```python
from Cython.Shadow import *
```

**Comment by erictraut** [comment]
The cython package includes a stub file `Shadow.pyi` that defines the symbols a static type checker sees. This stub file includes `int` and `char` but does not include `cfunc`. If you think that there are symbols that should be defined in this stub file but are not, please work with the maintainers of cython to add them.

Pyright is working correctly here given the type information provided to it, so I'm closing the issue.
